lib/cash-out/cash-out-atomic.js

```diff
@@ -9,6 +9,8 @@ const helper = require('./cash-out-helper')
 const cashOutActions = require('./cash-out-actions')
 const cashOutLow = require('./cash-out-low')
 
+const notifier = require("../notifier/index")
```

Comment:
Use single quotes here as well

```suggestion
const notifier = require('../notifier')
```